setenv.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+freq_file_name="/var/nebra/envvars/FREQ"
+variant_file_name="/var/nebra/envvars/VARIANT"
+
+# Create necessary folder if it doesn't exist
+if [ ! -d "/var/nebra/envvars" ]; then
+    mkdir "/var/nebra/envvars"
+fi
+
+#FREQ
+if [ -z ${FREQ+x} ]; then
+    if [ -f ${freq_file_name} ]; then
+        FREQ=$(<${freq_file_name});
+        export FREQ=${FREQ}
+        echo "FREQ is set as ${FREQ}";
+    else
+        echo "Error: Cannot set FREQ variable.";
+    fi
+else
+    echo "FREQ variable is already set as ${FREQ}.";
+    if [ ! -f ${freq_file_name} ]; then
+        echo "FREQ parameter file doesn't exist. Setting it as ${FREQ} now.";
+        echo "${FREQ}" > ${freq_file_name}
```

Comment:
may be reading from file can be a function. That way we can reuse it at four different places. we can also avoid always writing it.

Reply:
Makes sense.

Reply:
Done.